xdsl/dialects/mpi.py

```diff
@@ -230,9 +230,9 @@ def get(
             SSAValue | Operation | Sequence[SSAValue | Operation]
         ] = []
         if send_buffer is None:
-            operands_to_add = [recv_buffer, count, datatype, []]
+            operands_to_add = [[], recv_buffer, count, datatype]
```

Comment:
Why not pass None?

Reply:
Yeah, although I have a slight preference towards None, since it communicates that it's optional and not variadic

Reply:
I see.

Reply:
This is a nit, I don't feel strongly about this :)